backend/src/hmc-api/files.ts

```diff
@@ -96,6 +96,30 @@ export function parseCourse(data: string) {
     return courses;
 }
 
+export function parseCourseBoomi(data: string): {
+    code: string;
+    title: string;
+    campus: string;
+    description: string;
+}[] {
```

Comment:
I say define and export a separate named type for this
```typescript
export interface BoomiCourse {
  code: string;
  // ...
}
```
and use that in the function signature. since it's likely a type that'll get used elsewhere as well.

Reply:
generally it's useful to export any type that is slightly non-trivial; anonymous/inline types are really (ideally imo) only for small short things.